dask/bag/tests/test_bag.py

```diff
@@ -1187,6 +1199,12 @@ def test_repartition_names():
     assert b is c
 
 
+def test_repartition_input_errors():
+    with pytest.raises(ValueError):
+        bag = db.from_sequence(range(10))
+        bag.repartition(npartitions=5, partition_size="5MiB")
```

Comment:
Let's make sure that we have an additional test that verifies size on some more complex data types, like maybe a list of numpy arrays.  This will probably fail with `sys.getsizeof` but hopefully be ok with `dask.sizeof.sizeof`

Reply:
I added a test to this effect that fails with `sys.getsizeof` and passes with `dask.sizeof.sizeof` locally. However, on the Windows CI builds, the new test fails (but the test appears to pass on the `continuous-integration/travis-ci` build).  Is there a cheap way to debug Windows CI failures without waiting for the full CI build to fail and without setting up a Windows VM?